### ISSUE TITLE
Support for Account Moving from/to Takahe

### ISSUE THREAD

**Issue by tobru** [jointakahe/takahe#385]
Support for account moving, e.g. from Mastodon to Takahe (or the other way around).

See [Mastodon docs "Moving or leaving accounts"](https://docs.joinmastodon.org/user/moving/) for the possibilities Mastodon offers in this regard.

Priority (for me) would be moving from Mastodon _to_ Takahe.

**Comment by andrewgodwin** [comment]
Yup, this is on our roadmap for approaching 1.0, though some initial work has already started.

**Comment by everton137** [comment]
I see it's not possible to export posts from Mastodon. Has anyone checked how difficult it could be to import posts on a Takahe instance? I am running a Mastodon instance, but I would rather run a Takahe one, since I work with Python. I still haven't found time to test Takahe tho.

**Comment by gerwitz** [comment]
I'd be interested in a database import for Akkoma, as well, but am more concerned with triggering other instances to migrate followers.

**Comment by astrojuanlu** [comment]
Would Takahe -> Takahe migration be implemented first? Or is that already possible?

**Comment by andrewgodwin** [comment]
It would be at the same time - I'd want to implement the account moving system that's already out there, so it would just let you move to/from anything else that supports it.

**Comment by tobru** [comment]
I'm also wondering the other way around: How could a process look like to migrate from Takahe to any other Fediverse software, let's say GoToSocial?

**Comment by andrewgodwin** [comment]
It's the same system - one of the reasons I have not done another release yet is that I've implemented inbound migration, but not yet implemented outbound, and it feels bad to have one and not the other.

**Comment by ellieayla** [comment]
Regarding exporting posts from mastodon, the Export Archive feature certainly includes them... https://docs.joinmastodon.org/user/moving/#export - very little tooling actually supports importing this archive though, not even mastodon itself. Supporting backfill-import of the archive would be very cool, though perhaps not within the same issue as the online account migration.
